src/DateRangePicker/test/CalendarMonthDropdownSpec.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import CalendarContext from '../../Calendar/CalendarContext';
+import MonthDropdown from '../../Calendar/MonthDropdown';
+import { DateUtils } from '../../utils';
+
+describe('DateRangePicker-MonthDropdown', () => {
+  it('Should output year and month of previous and current year', () => {
+    const { getAllByRole } = render(
+      <CalendarContext.Provider
+        value={{
+          date: new Date(),
+          locale: {},
+          isoWeek: false
+        }}
+      >
+        <MonthDropdown show limitStartYear={1} limitEndYear={1} />
+      </CalendarContext.Provider>
+    );
+
+    const currentYear = DateUtils.getYear(new Date());
+    const previousYear = currentYear - 1;
+    expect(getAllByRole('row', { hidden: true })).to.be.lengthOf(2);
+    expect(getAllByRole('rowheader', { hidden: true })[0].innerText).to.be.eq(
+      previousYear.toString()
+    );
+    expect(getAllByRole('rowheader', { hidden: true })[1].innerText).to.be.eq(
+      currentYear.toString()
+    );
```

Comment:
Let me confirm this. Does it mean that `limitStartYear={1}` results in the earliest year be 1 year before today, while `limitEndYear={1}` results in the latest year be just current year? If so, I don't think it's consistent.

Reply:
Hi, yes, you're right and yes, it's not consistent.  The limitStartYear={1} results in the earliest year that is 1 year before current, while limitEndYear={1} results in the current year. The problem is that I only propagate the props to existent `MonthDropdown`-API. Should I change this API? I think it would mean a breaking change for people who already use MonthDropdown.